google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java

```diff
@@ -154,7 +155,7 @@ public void run() {
                 DebugTarget targetSelection = null;
 
                 if (debuggees == null || debuggees.getDebuggees() == null || debuggees.getDebuggees().isEmpty()) {
-                  disableTargetSelector();
+                  disableTargetSelector(new NoDebuggableModuleException());
```

Comment:
In a similar change I've decided to just create 2 methods, one with a String parameter and the other delegating to it after figuring out the message. I also don't feel strongly about this usage of Exception.